best-practices.md

```diff
@@ -282,3 +282,40 @@ training process to find the "best model". This field could also be used to indi
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `torch.save`                                     | A [serialized python pickle object][pytorch-save] (i.e.: `.pt`) which can represent a model or state_dict.                                                                                                                                                                                                                                  |
+| `torch.jit.save`                                 | A [`TorchScript`][pytorch-jit-script] model artifact obtained with one or more of the graph export options Torchscript Tracing and Torchscript Scripting.                                                                                                                                                                                   |
+| `torch.export.save`                              | A model artifact storing an [ExportedProgram][exported-program] obtained by [`torch.export.export`][pytorch-export] (i.e.: `.pt2`).                                                                                                                                                                                                         |
+| `tf.keras.Model.save`                            | Saves a [.keras model file][keras-model], a unified zip archive format containing the architecture, weights, optimizer, losses, and metrics.                                                                                                                                                                                                |
+| `tf.keras.Model.save_weights`                    | A [.weights.h5][keras-save-weights] file containing only model weights for use by Tensorflow or Keras.                                                                                                                                                                                                                                      |
+| `tf.keras.Model.export(format='tf_saved_model')` | TF Saved Model is the [recommended format][tf-keras-recommended] by the Tensorflow team for whole model saving/loading for inference. See this example to [save and load models][keras-example] and the docs for [different save methods][keras-methods] in TF and Keras. Also available from `keras.Model.export(format='tf_saved_model')` |
```

Comment:
I understand the idea of being explicit, but I wonder if that name makes it useful. 

For example, if the intention is to help create search/filter queries to find models that match this artifact type, the query definition becomes much harder to define because there will most probably be a need to escape the quotes, parenthesis and equal symbols all employed by most query languages. Wouldn't `tf.keras.Model.export` alone` be enough to hint about this use, since the `tf.` prefix is used anyway? 

I preferably would avoid hinting users toward writing definitions that look too much like directly executable code. It is preferable if there is just enough information to know which modules/functions to import, but parameters and calling them must still be done explicitly by "connecting the dots".


See also other comment about the links. Maybe not the right ones for what is described?

Reply:
I agree since the tf. prefix is included, will change